From 3308d4f133231b5d601c13c516c0ed9a5c23b5ed Mon Sep 17 00:00:00 2001
From: Gowtham Anandha Babu <gowtham.anandha.babu@intel.com>
Date: Fri, 9 Aug 2024 11:50:20 +0000
Subject: [PATCH] Migrate HIDL Software Gatekeeper to AIDL

When Trusty is false in mixin, use Software Gatekeeper
aidl service

Tests Done:
1. Flash the image Bare metal or CiV
2. Image is booting and service is running

Tracked-On: OAM-123194
Signed-off-by: Aman Bhadouria <aman.bhadouria@intel.com>
---
 groups/tee/false/product.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/groups/tee/false/product.mk b/groups/tee/false/product.mk
index 3153c11..33630cc 100644
--- a/groups/tee/false/product.mk
+++ b/groups/tee/false/product.mk
@@ -1,5 +1,5 @@
 PRODUCT_PACKAGES += \
-    android.hardware.gatekeeper-service.software \
+    com.android.hardware.gatekeeper.nonsecure \
     android.hardware.security.keymint-service
 
 #PRODUCT_PACKAGES += \
-- 
2.45.2

